src/transform/template_transform.rs

```diff
@@ -45,16 +43,21 @@ fn get_reversed_arguments(tpl: &Tpl) -> Vec<Expr> {
     let mut arguments = Vec::new();
     let mut index = 0;
     let empty_quasi = JsWord::from("");
+    let mut last_skipped = false;
     for quasi in &tpl.quasis {
         let value = quasi.cooked.clone();
         if value.is_none() || value.unwrap() == empty_quasi {
             if !quasi.tail {
                 let expr = &*tpl.exprs[index];
                 arguments.push(expr.clone());
                 index += 1;
+                last_skipped = true;
+            }
+            if !quasi.tail || !last_skipped {
```

Comment:
so this means if either current or previous iteration `quasi.tail` is false, correct ?